src/Makefile.global.in

```diff
@@ -598,6 +598,13 @@ TAS         = @TAS@
 
 ##########################################################################
 #
+# Link to ZMQ
+
+LIBS := -lzmq $(LIBS)
```

Comment:
I've created a ticket for this and will do it since I'm creating the new packaging containers right now.